src/components/Homepage/Homepage-cards.js

```diff
@@ -0,0 +1,41 @@
+import '../../css/home-page.css';
+import spaHomePage from '../../assets/spa-homepage.jpg';
+import diningHomePage from '../../assets/dining-homepage.jpg';
+
+const HomePageCards = () => {
```

Comment:
there is no need to create explicit components for homepage probably should make a card component using your component library and then if want to change or extend the styling if the card component then can do that from app.css file., this way there will be less files and less code,